include/sharg/validators.hpp

```diff
@@ -583,27 +583,52 @@ class output_file_validator : public file_validator_base
      * \{
      */
 
-    //!\copydoc sharg::input_file_validator::input_file_validator()
-    output_file_validator() : output_file_validator{output_file_open_options::create_new} {}
-
-    output_file_validator(output_file_validator const &) = default;             //!< Defaulted.
-    output_file_validator(output_file_validator &&) = default;                  //!< Defaulted.
+    output_file_validator() = default; //!< Defaulted.
+    output_file_validator(output_file_validator const &) = default; //!< Defaulted.
+    output_file_validator(output_file_validator &&) = default; //!< Defaulted.
     output_file_validator & operator=(output_file_validator const &) = default; //!< Defaulted.
-    output_file_validator & operator=(output_file_validator &&) = default;      //!< Defaulted.
+    output_file_validator & operator=(output_file_validator &&) = default; //!< Defaulted.
     virtual ~output_file_validator() = default; //!< Virtual Destructor.
 
     /*!\brief Constructs from a given overwrite mode and a list of valid extensions.
      * \param[in] mode A sharg::output_file_open_options indicating whether the validator throws if a file already
-                       exists.
+     *                 exists.
      * \param[in] extensions The valid extensions to validate for.
      */
-    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> extensions = {})
-        : file_validator_base{}, mode{mode}
+    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> const & extensions)
+        : open_mode{mode}
     {
         file_validator_base::extensions_str = detail::to_string(extensions);
         file_validator_base::extensions = std::move(extensions);
     }
 
+    /*!\brief Constructs from a given overwrite mode and a parameter pack of valid extensions.
+     * \param[in] mode A sharg::output_file_open_options indicating whether the validator throws if a file already
+     *                 exists.
+     * \param[in] extensions Parameter pack representing valid extensions. std::string must be constructible from each
+     *                       argument. The pack may be empty ( → all extensions are valid).
+     */
+    explicit output_file_validator(output_file_open_options const mode, auto && ...extensions)
+        requires ((std::constructible_from<std::string, decltype(extensions)> && ...))
+        : output_file_validator{mode, std::vector<std::string>{std::forward<decltype(extensions)>(extensions)...}}
+    {}
+
+    /*!\brief Constructs from a list of valid extensions.
+     * \param[in] extensions The valid extensions to validate for.
+     */
+    explicit output_file_validator(std::vector<std::string> const & extensions)
+        : output_file_validator{output_file_open_options::create_new, extensions}
+    {}
```

Comment:
If I use this constructor and an empty list, are all extensions valid then?

Reply:
It's the same, because we do `file_validator_base::extensions = std::move(extensions);` and `file_validator_base::extensions` is empty by default.

I added explicit tests :)